tests/conftest.py

```diff
@@ -74,58 +74,74 @@ def base_route(api_version, gordo_project, gordo_name):
     return f"/gordo/{api_version}/{gordo_project}/{gordo_name}"
 
 
+@pytest.fixture(scope="session")
+def model_collection_directory(gordo_project: str):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        collection_dir = os.path.join(tmp_dir, gordo_project)
+        os.makedirs(collection_dir, exist_ok=True)
+        yield collection_dir
+
+
+@pytest.fixture(scope="session")
+def config_str(gordo_name: str, sensors: List[SensorTag]):
+    """
+    Fixture: Default config for testing
+    """
+    yield f"""
+            machines:
+              - dataset:
+                  tags:
+                    - {sensors[0].name}
+                    - {sensors[1].name}
+                    - {sensors[2].name}
+                    - {sensors[3].name}
+                  target_tag_list:
+                    - {sensors[0].name}
+                    - {sensors[1].name}
+                    - {sensors[2].name}
+                    - {sensors[3].name}
+                  train_start_date: '2019-01-01T00:00:00+00:00'
+                  train_end_date: '2019-10-01T00:00:00+00:00'
+                  asset: asgb
```

Comment:
No strong feelings here, but should we do one of three things here:
1. Change this to `{sensors[0].asset}` <- assuming all other tags are the same asset, which I guess is pretty safe.
2. Update things like `tags` elements to: `{dict(sensors[0]._asdict())}` which would then ensure each element had an asset defined and delete this `asset` param.
3. Leave it, because I guess it doesn't _really_ matter. :sweat_smile:

Reply:
Currently, all of those sensors have no asset set I believe, it's getting set here in this config, so 1. and 2. would break in some cases when normalize_sensors is called -- without patching the asset somehow-- I think.